datacube_alchemist/cli.py

```diff
@@ -212,16 +213,27 @@ def redrive_to_queue(queue, to_queue, limit):
 
     count = 0
 
-    for message in messages:
-        response = alive_queue.send_message(MessageBody=message.body)
-        if response["ResponseMetadata"]["HTTPStatusCode"] == 200:
-            message.delete()
-            count += 1
-            if limit and count >= limit:
-                break
-        else:
-            _LOG.error(f"Unable to send message {message} to queue")
-    _LOG.info(f"Completed sending {count} messages to the queue")
+    count_messages = dead_queue.attributes.get("ApproximateNumberOfMessages")
+
+    if count_messages == 0:
```

Comment:
It's not worth trying anyway? It isn't called 'ApproximateNumberOfMessage' for nothing. :)

Reply:
Maybe you're right. Doesn't make much difference.

Reply:
Agreed :)